common/test/acceptance/tests/helpers.py

```diff
@@ -234,6 +234,24 @@ def element_has_text(page, css_selector, text):
     return text_present
 
 
+def assert_event_emitted_num_times(event_collection, event_name, event_time, event_user_id, num_times_emitted):
+    """
+    Tests the number of times a particular event was emitted.
+    :param event_collection: MongoClient instance to query.
+    :param event_name: Expected event name (e.g., "edx.course.enrollment.activated")
+    :param event_time: Latest expected time, after which the event would fire (e.g., the beginning of the test case)
```

Comment:
It seems like `event_user_id` and `num_times_emitted` should also be included in this docstring. They're self-documenting, but if parameters are being listed in the docstring, I think it'd be good to be complete.